components/learn/boxes/Boxes.js

```diff
@@ -155,13 +155,23 @@ export default function Boxes(props) {
 
                       <div className={styles.content}>
                         <p className={styles.title}>
-                          <a href={`${prefix}/learn/asyncapi-tool`} className={styles.titleLink}>
-                            FHIR tool
+                          <a href={`${prefix}/learn/health-tool`} className={styles.titleLink}>
+                            Health tool (FHIR/HL7)
                           </a>
                         </p>
                         <p className={styles.description}>Details of the Ballerina FHIR profile to client and stub generation tool.</p>
```

Comment:
Reword to `Details of the FHIR profile to client and stub generation tool of Ballerina` for clarity. Otherwise it seems like `Ballerina FHIR profile` goes as one thing.

Reply:
Updated.